lang/circom/fyi.list

```diff
@@ -0,0 +1,3 @@
+semgrep-grammars/src/tree-sitter-circom/LICENSE
```

Comment:
This LICENSE file is missing in your tree-sitter-circom repository, which prevents me to run
the ./release circom script to populate the https://github.com/semgrep/semgrep-circom
Please add this file, update the submodule, and make a PR, so I can then run ./release circom

Reply:
Thank you, fixed in PR: https://github.com/semgrep/ocaml-tree-sitter-semgrep/pull/495